mapscaleview/src/main/java/com/github/pengrad/mapscaleview/MapScaleView.java

```diff
@@ -43,6 +43,8 @@ public MapScaleView(Context context, AttributeSet attrs, int defStyleAttr) {
         if (viewConfig.isMiles) {
             scaleType = ScaleType.MILES_ONLY;
         }
+
+        drawer.setExpandLeftEnabled(viewConfig.isExpandLeft);
```

Comment:
Let's pass it in constructor.
`setExpandLeftEnabled` method will be unusable after that, so you can remove it or make it available for runtime changes via `view.setExpandRtlEnabled` same with `setOutlineEnabled`

Reply:
I see.
Let's do it like setOutlineEnabled.